views/index.pug

```diff
@@ -0,0 +1,22 @@
+extends layout.pug
+
+block head
+	header
+		.logo
+			h1 <br>Styleguide
+			h4 Git
```

Comment:
h4 is not inside of h1, I've not given any indentation. anyways this is a temp for the git icon